File: Linkedlist/linkedlist.py
class node:
    def __init__(self,data):
        self.data=data
        self.next=None
        
        

class linkedlist(node):
    def __init__(self):
        self.head=None
    def insert(self,newnode):
        if(self.head==None):
            self.head=newnode
        else:
            tempnode=self.head
            while(True):
                if(tempnode.next==None):
                    break
                tempnode=tempnode.next
            tempnode.next=newnode
            
    def print_list(self):
        temp=self.head
        p=''
        while(temp!=None):
            l=str(temp.data)
            p+=l+'->'
            temp=temp.next
        print(p)    
        
        
firstnode=node("man") 
linked=linkedlist()
linked.insert(firstnode)
second=node("lij") 
linked.insert(second)
third=node("paulo") 
linked.insert(third)
linked.print_list()
    
        
            
        

File: Linkedlist/linkedlist.java
import java.util.*;


class Node{
    int data;
    Node next;
    
    
}



class Link{
    Node head = null;
    public void insert(int data){
        
     Node n = new Node();
     n.data=data;
     n.next=null;
        if(head==null){
            
            
            head=n;
        }
        else{
            
            Node temp = head;
            while(temp.next!=null){
                
                
                temp=temp.next;
                
            }
            temp.next=n;
        }
        
    }
    Node v=head;
    public void dis(){
       
        while(v!=null){
            System.out.println(Integer.toString(v.data));
            v=v.next;
        }
    }
    
}


public class HelloWorld{

     public static void main(String []args){
        
        
        Link l =new  Link();
        l.insert(7);
        l.insert(10);
        l.insert(190);
        l.dis();
      //  System.out.println(Integer.toString(l.head.next.data));
        
        
     }
}
